### PR TITLE
Add pre-genesis

### PR DESCRIPTION
**Pull request by albertandrejev**

### CODE REVIEW

File: quark/pre-genesis.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        }
+      }
+    },
+    "interchainqueries": {
+      "params": {
+        "query_submit_timeout": "43200",
+        "query_deposit": [
+          {
+            "denom": "untrn",
+            "amount": "1000000"
```
Comment (sotnikov-s) [inline quark/pre-genesis.json:170]: isn't it too little? will it be profitable for
1) the owner to delete query and get his funds back or
2) other users to clean up chain from forgotten queries

given the fee taken for delete query calls?

Comment (foxpy) [reply quark/pre-genesis.json:170]: well, you can have fees as little as 0.01-0.02 NTRN, and deposit size is 1 NTRN.

Comment (sotnikov-s) [reply quark/pre-genesis.json:170]: I haven't tried this flow. Have you? Is the fee so little for a delete call?

Comment (albertandrejev) [reply quark/pre-genesis.json:170]: fee usually low enough and in this case we need it as PoC. We can increase this value via Gov Proposal

Comment (foxpy) [reply quark/pre-genesis.json:170]: > Is the fee so little for a delete call?

It is. Think of it as of withdrawing staking rewards. e.g. withdrawing 1 ATOM.

I don't see any problems with 1 NTRN deposit, it is not very expensive but still enough to stop people from spamming registered queries.

**Review by foxpy — APPROVED** [review]
LGTM, though I am not experienced in creating genesises, so, reviews from other teammates are very appreciated!

**Review by sotnikov-s — DISMISSED** [review]

File: quark/pre-genesis.json (hunk 1 above)
Comment (foxpy) [reply quark/pre-genesis.json:170]: Now it is set to 10 NTRN, even better in this regard.

**Review by foxpy — APPROVED** [review]

**Pull request merged**